### PR TITLE
feat: suggest drill-down & strict templates

### PR DESCRIPTION
**Pull request by anbalase**
image

### CODE REVIEW

File: projects/angular/components/ui-suggest/src/ui-suggest.component.ts [hunk 1]
```diff
@@ -133,6 +137,19 @@ export class UiSuggestComponent extends UiSuggestMatFormFieldDirective
...
+    /**
+     * Configure if the component allows expandable items
+     *
+     */
+    @HostBinding('class.drill-down')
+    @Input()
+    get drillDown() {
+        return this._drillDown;
+    }
+    set drillDown(value) {
```
Comment (CalinaCristian) [inline projects/angular/components/ui-suggest/src/ui-suggest.component.ts:150]: shouldn't we do value: Boolean so we enforce in strict templates on the consumer the type?

File: tsconfig.json [hunk 2]
```diff
@@ -76,5 +76,8 @@
                 "./projects/angular/pipes/src/public_api.ts"
             ],
         }
-    }
+    },
+    "angularCompilerOptions": {
+        "strictTemplates": true,
```
Comment (CalinaCristian) [inline tsconfig.json:81]: <3

File: projects/angular/components/ui-grid/src/ui-grid.component.html [hunk 3]
```diff
@@ -544,11 +544,7 @@
                         [searchSourceFactory]="column.searchableDropdown!.searchSourceFactory"
                         [value]="column.searchableDropdown!.value ? [column.searchableDropdown!.value!] : []"
                         [disabled]="column.searchableDropdown!.disabled || disabled"
-                        [placeholder]="column.title"
```
Comment (CalinaCristian) [inline projects/angular/components/ui-grid/src/ui-grid.component.html:542]: 🤣

**Review by CalinaCristian — APPROVED** [review]
🚀

**Review by alexdudescu — APPROVED** [review]
lgtm, did not check functionally

**Pull request merged**